sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -159,25 +150,27 @@
     "frequency/@units": "kHz",
     "associated_beam": "/entry/instrument/beam_pump"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_pump]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_pump]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_wavelength": "@attrs:metadata/scientificMetadata/Laser/wavelength/value",
```

Comment:
This has the unit inside the "value" field in the scicat file, needs to be fixed there.

Reply:
made a workaround